bin/PurgeHeldJobs.py

```diff
@@ -173,4 +173,4 @@ def handle_hold_event(event):
             time.sleep(5)
         except Exception as e:
             slack_client.ee2_reaper_failure(endpoint=ee2_endpoint)
-            sys.exit(f"{e}")
```

Comment:
is there any concern over keeping the reaper running when there's a legitimate error here?

Reply:
No, the reaper should never end